libraries/path_helpers.rb

```diff
@@ -0,0 +1,24 @@
+# Encoding: utf-8
+
+module PathHelpers
```

Comment:
Yeah, that's probably a good idea. I was thinking of the modules organizationally at first, which that doesn't quite work now that I have two helper modules.